### PR TITLE
Fix CLI reference page

### PR DESCRIPTION
**Pull request by cdrage**
The CLI reference page wasn't rendering correctly. This fixes it (ran
through pandoc converter...)

### CODE REVIEW

**Comment by kadel** [comment]
```
evel=error msg="Terraform does not automatically rollback in the face of errors."
level=error msg="Instead, your Terraform state file has been partially updated with"
level=error msg="any resources that successfully completed. Please address the error"
level=error msg="above and apply again to incrementally change your infrastructure."
level=error
level=error
```


/test e2e-scenarios

File: docs/cli-reference.adoc [hunk 1]
```diff
@@ -1,4 +1,5 @@
-= Overview of the `odo` CLI Structure
+= Overview of the Odo (OpenShift Do) CLI Structure
```
Comment (kadel) [inline docs/cli-reference.adoc]: agreed style to for `odo` is to use it in monospace like `odo` if possible or always in lowercase (except when it is at the beginning of the sentence)

File: docs/cli-reference.adoc [hunk 2]
```diff
@@ -46,110 +54,119 @@ List of Commands
...
+        list --app --component --context --output --project : List URLs
+    utils : Utilities for terminal commands and modifying Odo configurations (terminal)
         terminal : Add Odo terminal support to your development environment
     version --client : Print the client version information
-    watch --app --delay --ignore --project : Watch for changes, update component on change
+    watch --app --context --delay --ignore --project : Watch for changes, update component on change
 ----
 
+[[app]]
 app
```
Comment (mik-dass) [inline docs/cli-reference.adoc]: Should we add `[options]` after app and the other commands below?

**Comment by kadel** [comment]
/approve
but this needs to be fixed properly in https://github.com/openshift/odo/issues/1565

**Comment by kadel** [comment]
/retest 
```
level=warning msg="Found override for ReleaseImage. Please be warned, this is not advised"
level=info msg="Consuming \"Install Config\" from target directory"
level=info msg="Creating infrastructure resources..."
level=error
level=error msg="Error: Error applying plan:"
level=error
level=error msg="1 error occurred:"
level=error msg="\t* module.vpc.aws_lb.api_internal: 1 error occurred:"
level=error msg="\t* aws_lb.api_internal: timeout while waiting for state to become 'active' (last state: 'provisioning', timeout: 10m0s)"
level=error
level=error
level=error
level=error
level=error
level=error msg="Terraform does not automatically rollback in the face of errors."
level=error msg="Instead, your Terraform state file has been partially updated with"
level=error msg="any resources that successfully completed. Please address the error"
level=error msg="above and apply again to incrementally change your infrastructure."
level=error
level=error
level=fatal msg="failed to fetch Cluster: failed to generate asset \"Cluster\": failed to create cluster: failed to apply using Terraform"

```

**Comment by kadel** [comment]
minor fixes. document can't be in worse shape than it is ;-)
/lgtm

**Comment by kadel** [comment]
docs change

/override ci/prow/e2e-scenarios 
/continuous-integration/travis-ci/pr

**Pull request merged**